src/Mysql.php

```diff
@@ -30,11 +30,13 @@ class Mysql
      * @param $database
      * @param int $fetchMode
      * @param string $charset
+     * @param array $pdo_options
      * @param array $options
      *
      * @throws MysqlException
      */
-    public function __construct($host, $user, $password, $database, $fetchMode = \PDO::FETCH_ASSOC, $charset = 'utf8', array $options = [])
+    public function __construct($host, $user, $password, $database, $fetchMode = \PDO::FETCH_ASSOC,
+                                $charset = 'utf8', array $pdo_options = array(), array $options = [])
```

Comment:
There is a `PDO::ATTR_PERSISTENT` attribute in PDO class which can create a persistent connection. You can find it in this page: `http://jp2.php.net/manual/zh/features.persistent-connections.php`. I just read it and I found that this attribute only works in the cgi env.

It will be better that users can set their own POD attributes.